model/riscv_sys_regs.sail

```diff
@@ -826,6 +881,15 @@ bitfield Vtype  : xlenbits = {
 }
 register vtype : Vtype
 
+//Returns the PMM bits value depends on the Privilege mode
+function get_pmm() -> bits(2) = {
+  match cur_privilege {
+    Machine => mseccfg[PMM],
+    Supervisor => menvcfg[PMM],
+    User => if extensionEnabled(Ext_S)  then senvcfg[PMM] else menvcfg[PMM],
```

Comment:
Extra space before `then`. You can also omit the outer `{}` on this function.

Reply:
Done